src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -544,7 +889,26 @@ protected Solution CreateOrOpenSolution(bool nullPaths = false)
             return storage;
         }
 
-        private class MockPersistentStorageLocationService : IPersistentStorageLocationService
+        internal IChecksummedPersistentStorage GetStorageFromKey(
+            Workspace workspace, SolutionKey solutionKey, IPersistentStorageFaultInjector faultInjectorOpt = null)
+        {
+            // If we handed out one for a previous test, we need to shut that down first
```

Comment:
We're sharing state between tests? This isn't dead code from something @sharwell cleaned up, is it?

Reply:
(this isn't blocking just curious)

Reply:
There's likely a better way to do this. However, the tests actually perform IO (since they're actually creating the actual sqlite DB on disk). So this is about proper creation/deletion of those files.